AcloConcurrency/AcloConcurrency/Collection/Cocktails/Views/CocktailsView.swift

```diff
@@ -0,0 +1,51 @@
+import SwiftUI
+
+struct CocktailsView: View {
+
+    private let ingredientName: String
+
+    @ObservedObject private var cocktailsViewModel =
+    CocktailsViewModel(filterService: FilterService())
```

Comment:
why you init it here?

Reply:
Should I unit in parent view in nav Link?